### PR TITLE
JDF-782: Replace add-user instructions and fix links to point to JBoss ddeveloper shared repository

### PR DESCRIPTION
**Pull request by sgilda**
@rafabene : In addition to the link, can you please make sure the credentials specified are also correct? 

@mareknovotny : I modified the create admin user as you suggested.

### CODE REVIEW

File: deltaspike-authorization/README.md [hunk 1]
```diff
@@ -39,7 +39,31 @@ If you have not yet done so, you must [Configure Maven](../README.md#configure-m
...
+This quickstart uses secured management interfaces and requires that you create the following `quickstartUser` application user to access the running application. 
+
+| **UserName** | **Realm** | **Password** | **Roles** |
+|:-----------|:-----------|:-----------|:-----------|
+| quickstartUser| ApplicationRealm | quickstartPwd! | guest |
+
+To add the application user, open a command prompt and type the following commands:
+
+        For Linux:
+             EAP_HOME/bin/add-user.sh -a -u quickuser -p quick-123
```
Comment (rafabene) [inline deltaspike-authorization/README.md]: Shouldn't the right one be : EAP_HOME/bin/add-user.sh -a -u 'quickstartUser' -p 'quickstartPwd1!' -g 'guest' ?

**Comment by sgilda** [comment]
@rafabene : Good catch. Thanks!

**Comment by mareknovotny** [comment]
@sgilda reworded text is great!

**Comment by sgilda** [comment]
@mareknovotny : Thanks!
@rafabene : Are you okay with this too?

**Comment by sgilda** [comment]
@rafabene : If you see any problems, I'll submit another pull. :-)

File: deltaspike-authorization/README.md [hunk 2]
```diff
@@ -39,7 +39,31 @@ If you have not yet done so, you must [Configure Maven](../README.md#configure-m
...
+This quickstart uses secured management interfaces and requires that you create the following `quickstartUser` application user to access the running application. 
+
+| **UserName** | **Realm** | **Password** | **Roles** |
+|:-----------|:-----------|:-----------|:-----------|
+| quickstartUser| ApplicationRealm | quickstartPwd1! | guest |
+
+To add the application user, open a command prompt and type the following commands:
+
+        For Linux:
+            EAP_HOME/bin/add-user.sh -a -u quickstartUser -p quickstartPwd1!
```
Comment (rafabene) [inline deltaspike-authorization/README.md:52]: I still miss the -g guest here. Sorry for the late response

Comment (sgilda) [inline deltaspike-authorization/README.md:52]: @rafabene : Argghhh. Very bad day! Submitted another pull. Sorry about that.

**Pull request merged**